dot-gen/src/main/java/com/wipro/ats/bdre/ldg/GetDotForTable.java

```diff
@@ -0,0 +1,126 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.wipro.ats.bdre.ldg;
+
+import com.wipro.ats.bdre.GetLineageNodeByColName;
+import com.wipro.ats.bdre.GetNodeIdForLineageRelation;
+import com.wipro.ats.bdre.lineage.LineageMain;
+import com.wipro.ats.bdre.md.dao.jpa.LineageNode;
+import com.wipro.ats.bdre.md.dao.jpa.LineageRelation;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by AshutoshRai on 1/20/16.
+ */
+public class GetDotForTable {
+    protected static final Logger LOGGER = LoggerFactory.getLogger(GetDotForTable.class);
+    private String dotString;
+
+    //fn() to get DOT through Column name & table name
+    public String dotGeneratorWithCol(String[] args) {
+        if(args.length !=2) {
+            LOGGER.info("Two arguments required: please Give both Column name and Table name");
+            System.exit(-1);
+        }
+        GetDotForTable getDotForTable = new GetDotForTable();
+        GetNodeIdForLineageRelation getNodeIdForLineageRelation = new GetNodeIdForLineageRelation();
+
+        List<LineageRelation> lineageRelationList;
+        List<LineageRelation> lineageRelationListForTargetNodes;
+
+        //get the nodes which have the same DISPLAY_NAME as that of the string(COLUMN NAME) provided
+
+        GetLineageNodeByColName getLineageNodeByColName;
+        LineageMain lineageMain = new LineageMain();
+        //src table name is provided by user
+        String srcTableName = args[1];
+        LOGGER.info("Generating DOT for the given Column Name: " + args[0]);
+        String targetTableName = null;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "targetTableName" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1481)